pkg/metrics/features/metrics.go

```diff
@@ -545,6 +547,20 @@ func NewMetrics(withDefaults bool) Metrics {
 			Subsystem: subsystemNP,
 			Name:      "host_network_policies_present",
 		}),
+
+		NPDNSIngested: metric.NewGauge(metric.GaugeOpts{
```

Comment:
And same here and with all other `*Ingested` metrics added in successive commits.